languages/csharp/concept-exercises/pattern-matching/.docs/after.md

```diff
@@ -0,0 +1,3 @@
+# After
+
+- [Tutorial on pattern matching](https://docs.microsoft.com/en-us/dotnet/csharp/pattern-matching)
```

Comment:
This document is also referred to in the `hints.md` file and does not really add much in terms of in-depth explanation. So I think I this can be removed, unless you found another document.